addons/invert-stage/userscript.js

```diff
@@ -0,0 +1,37 @@
+export default async function ({ addon, console, msg }) {
+  // INVERT STAGE ADDON BY @BATMANGREEN123 (MICHEALTHERATZ)
+  const vm = addon.tab.traps.vm;
+  const canvas = vm.renderer.canvas;
+
+  // I spent an HOUR fixing a bug until I found out I was MISSING this variable!!!!
+  // THANKS [REDACTED]!
+  let invert = false;
+
+  // CREATING THE BUTTON (Using material icons bc i'm lazy)
+  const img = document.createElement("img");
+  img.className = "invert-btn btn-container";
+  img.draggable = false;
+  img.src = addon.self.dir + "/contrast.svg";
+
+  // EVENT LISTENERS
+  img.addEventListener("click", () => invert = !invert);
+  addon.tab.displayNoneWhileDisabled(img);

```

Comment:
Dynamic disable/enable is not actually enabled in the `addon.json` file, so this doesn't do anything.